e2e/testcafe-devextreme/tests/chat/typingIndicator.ts

```diff
@@ -0,0 +1,113 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import Chat from 'devextreme-testcafe-models/chat';
+import { createUser, generateMessages } from './data';
+import url from '../../helpers/getPageUrl';
+import { createWidget } from '../../helpers/createWidget';
+import { testScreenshot } from '../../helpers/themeUtils';
+import { appendElementTo, insertStylesheetRulesToPage } from '../../helpers/domUtils';
+
+const CHAT_TYPINGINDICATOR_CIRCLE_CLASS = 'dx-chat-typingindicator-circle';
+
+fixture.disablePageReloads`ChatTypingIndicator`
+  .page(url(__dirname, '../container.html'));
+
+test('Chat: typing indicator with emptyview', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'Typing indicator with emptyview.png', { element: '#chat' });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await appendElementTo('#container', 'div', 'chat');
+  await insertStylesheetRulesToPage(`.${CHAT_TYPINGINDICATOR_CIRCLE_CLASS} { animation: none; }`);
```

Comment:
We turned off animation but on the etalons there are difference between them. Start position in material doesn't fit start position in fluent
<img width="180" alt="image" src="https://github.com/user-attachments/assets/8f55a428-1030-4ccb-8c68-9f4e159682d3"> 
<img width="210" alt="image" src="https://github.com/user-attachments/assets/00841643-0835-4ee3-b205-dff9527548c4">
If I assume right the circles should have the same sizes beetween themes

Reply:
That's right. But you check that PR the moment the tests are running, and the screenshots can only be updated after I get new ones.

Reply:
[37aabca7f1d6bc866ca3bb6e9c5eaab00e2a6ac1]